freespace2/freespace.cpp

```diff
@@ -6653,6 +6653,11 @@ int game_main(int argc, char *argv[])
 		return 0;
 	}
 
+	Script_system.RunInitFunctions();
```

Comment:
What's the purpose of this being duplicated from `game_init`?

Reply:
Oh rip lol. I'm in a meeting and did not fully read this. That's a copy/paste error. Removed.